packages/react/src/components/Accordion/Accordion.tsx

```diff
@@ -23,6 +25,7 @@ import {
   ShorthandRenderFunction,
   withSafeTypeForAs,
 } from '../../types'
+import { ContainerFocusHandler } from 'src/lib/accessibility/FocusHandling/FocusContainer'
```

Comment:
Convert this to relative import: `'../../lib/accessibility/FocusHandling/FocusContainer'`